web/src/components/tasks/business-formation/business/MainBusinessAddressNj.test.tsx

```diff
@@ -100,22 +100,22 @@ describe("<MainBusinessAddressNj />", () => {
     expect(screen.getByLabelText("Address zip code")).toBeInTheDocument();
   });
 
-  it("does NOT show inline errors when missing address1 as NJ", async () => {
+  it("does show inline errors when missing address1 as NJ", async () => {
     await getPageHelper({ addressLine1: "" });
     fireEvent.blur(screen.getByLabelText("Address line1"));
-    expect(screen.queryByText(Config.formation.fields.addressLine1.error)).not.toBeInTheDocument();
+    expect(screen.getByText(Config.formation.fields.addressLine1.error)).toBeInTheDocument();
   });
 
-  it("does NOT show inline errors when missing city as NJ", async () => {
+  it("does show inline errors when missing city as NJ", async () => {
     await getPageHelper({ addressMunicipality: undefined });
     fireEvent.blur(screen.getByTestId("addressMunicipality"));
```

Comment:
Logic has changed, getPageHelper fills out other address fields, however if one of the required fields are missing than it will be shown